README.md

```diff
@@ -0,0 +1,73 @@
+![image](https://img.shields.io/pypi/dm/discord-py-slash-command.svg)
+![image](https://img.shields.io/pypi/pyversions/discord-py-interactions.svg)
+![image](https://img.shields.io/pypi/v/discord-py-interactions.svg)
+![image](https://readthedocs.org/projects/interactionspy/badge/?version=latest)
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
+![image](https://discord.com/api/guilds/789032594456576001/embed.png)
+
+------------------------------------------------------------------------
+
+# Interactions.py - A Feature-rich Discord Bot Framework for Python
+
+A highly extensible, easy to use, and feature complete bot framework for Discord, `Interactions.py` is the culmination of years of experience in the Discord API and bot development. This library has been built from the ground up with community feedback and suggestions in mind, providing a modern and intuitive API for easy interaction with the Discord API in Python.
+
+## Key Features
+Interactions.py offers a range of features for building Discord bots with Python, including:
+- ✅ 100% coverage of the Discord API
+- ✅ Dynamic cache with TTL support
+- ✅ Modern and Pythonic API for easy interaction with Discord
+- ✅ Proper rate-limit handling
+- ✅ Feature parity with most other Discord API wrappers
+- ✅ Fully automated command synchronisation
+
+In addition to core functionality, interactions.py provides a range of optional extensions, allowing you to further customize your bot and add new features with ease.
+
+## Extensibility
+
+So the base library doesn't do what you want? No problem! With builtin extensions, you are able to extend the functionality of the library. And if none of those pique your interest, there are a myriad of other extension libraries available.
+
+Just type `bot.load("extension")`
+
+<details>
```

Comment:
Call me a complainer, but something about this whole section feels really off. I don't like how it's just all collapsible, and for some reason I feel like we don't really necessarily *need* to share this on the readme despite it being useful. Maybe this is just me, but we should break the extensibility off into somewhere else?

Willing to hear rationale on this

Reply:
It was used to prevent reader fatigue. Large blocks of text will "scare" people off from reading a document. Extensions are a low-priority part of the library for initial "sale" however they are still a key selling point. 

For someone higher level, such as us, it's unnecessary and just adds an extra step to reading - for a less experienced dev it is a valuable technique to keep them engaged 

By hiding it; by the point, someone reaches the part they will expand it - sunk cost kicks in and they keep reading.